Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,19 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+//Answer/Task-1
+console.log("Gryffindor House Members:");
+hogwarts.forEach(({firstName, lastName, house}) =>{
+
+  if (house === "Gryffindor") {
+    console.log(`${firstName} ,${lastName}`);
+  }
+});
+//Answer/Task-2
+console.log("Teachers with Pets:");
+hogwarts.forEach(({firstName, lastName, pet, occupation}) =>{
```

Comment:
If you don't use `occupation` variable inside of your forEach callback, why do you think you need it in destructuring?

Please, also pay attention to your code formatting:

Bad: =>{
Good: => {

Bad: } ,${
Good: }, ${